Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM node:8.15.0
 
-# Install VS Code's deps. These are the only two it seems we need.
+# Install VS Code's deps. These are the only two it seems we need. Install git for source control.
```

Comment:
Second sentence is in the wrong place (and extraneous imo)

Reply:
I agree, from old a commit I just added. Will remove.